README.md

```diff
@@ -1 +1,42 @@
-# Module-1---Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+> This project was done using Html and CSS
+
+
+## Done With
```

Comment:
- [ OPTIONAL] In my opinion it would be better if you use the template given [here](https://github.com/microverseinc/readme-template), as it's an update of this old one has more description and features